pom.xml

```diff
@@ -194,32 +194,32 @@
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>util-rabbitmq</artifactId>
-                <version>9.0.0-1293</version>
+                <version>10.0.0-US1009117-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>worker-api</artifactId>
-                <version>9.0.0-1293</version>
+                <version>10.0.0-US1009117-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>worker-configs</artifactId>
-                <version>9.0.0-1293</version>
+                <version>10.0.0-US1009117-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>worker-queue-rabbit</artifactId>
-                <version>9.0.0-1293</version>
+                <version>10.0.0-US1009117-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>worker-store-http</artifactId>
-                <version>9.0.0-1293</version>
+                <version>10.0.0-US1009117-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>worker-store-mem</artifactId>
-                <version>9.0.0-1293</version>
+                <version>10.0.0-US1009117-SNAPSHOT</version>
             </dependency>
```

Comment:
This causing build failure in https://sou-jenkins2.swinfra.net/job/JobService/view/Developer/job/JobService%7Eworker-taskstowing%7EUS1009117-2%7ECI/1/ due to taking the nearest version of workerframework